source/includes/fact-mongodump-mongorestore-use-cases.rst

```diff
@@ -0,0 +1,9 @@
+:binary:`~bin.mongoimport` and :binary:`~bin.mongoexport` are not
+intended for the following use cases:
+
+- backing up and restoring data
+- converting data to a different format and then back to the original
+  format
+- sending data to a new destination and then back to the source
```

Comment:
```suggestion
- sending data from a MongoDB cluster to another format and then back to the cluster
```

Reply:
I'm hesitant to make this change because the wording does not sound precise - to my understanding, data is either A) converted to a new format or B) sent to a new destination but I'm not sure if the phrase "sent to a format" is accurate

Reply:
"converting data from a MongoDB cluster to another format and then back to the cluster" is fine with me too